### DISCUSSION TITLE
Table Tree Work in redux data

### DISCUSSION THREAD

**Discussion by YuriyAlekseev9567** [rsuite/rsuite#2450] [General]
Tree table dont work with data from redux store. because it is immutable.
Why you write something inside the row?
![Снимок экрана 2022-04-14 042738](https://user-images.githubusercontent.com/84084334/163296457-9ce522e3-3126-4a06-a3c8-124475c9b2bb.png)

**Comment by SevenOutman** [comment]
I believe we should improve the implementation here.
/cc @simonguo 
/cc @superman66 This applies to pickers as well

**Comment by simonguo** [comment]
This is done to take into account the rendering performance issues caused by object depth comparison when Table and Tree are used in `virtualized` scenarios.

Of course, it is not an excellent solution, and we need to find ways to improve it in the future.
